aten/src/ATen/CMakeLists.txt

```diff
@@ -301,14 +301,24 @@ IF(USE_FBGEMM_GENAI)
 
       # Add additional HIPCC compiler flags for performance
       set(FBGEMM_GENAI_EXTRA_HIPCC_FLAGS
-        -mllvm
-        -amdgpu-coerce-illegal-types=1
```

Comment:
> clang++ and amdclang++ provided by therock versions 7.10 and 7.11 failed to
build Pytorch for a following error:
>
> clang++: error: unknown argument: '-amdgpu-coerce-illegal-types=1'

Are these the errors we were seeing for https://github.com/ROCm/TheRock/issues/2056 ? We've had FBGEMM_GENAI _unconditionally disabled_ in TheRock due to that issue (cc @cthi )